### ISSUE TITLE
xxx muscle terms

### ISSUE THREAD

**Issue by slaulederkind** [obophenotype/uberon#1295]
Why is it that the terms for skeletal muscles end with "muscle" sometimes and sometimes not?  For example these sibling terms:

flexor cruris lateralis muscle +  
flexor digitorum profundus 
flexor hallucis brevis muscle 
flexor hallucis longus 
gastrocnemius +

**Comment by cmungall** [comment]
Good Q. We always suffix where there is potential for ambiguity, but for 
longer less ambiguous names we often contract. This seems generally to 
be the case for wikipedia, other AOs, ...

I defer to @rdruzinsky here

**Comment by RDruzinsky** [comment]
I don't think that it matters. Since they are all muscles, I don't think that "muscle" must be in the name. We should just decide one way or the other and make them all consistent.

**Comment by RDruzinsky** [comment]
Upon further reflection, I think that we should eliminate "muscle" from the
names of the muscles.

**Comment by cmungall** [comment]
What about cases where this would create ambiguity, e.g. pterygoid?

**Comment by fbastian** [comment]
Plus it's bothersome to have to look at the parent to determine what a term is about.

**Comment by cmungall** [comment]
Good point.

A compromise could be to treat 'muscle' as redundant and exclude it, if the label already has a muscle type in like like 'flexor'.

**Comment by RDruzinsky** [comment]
Okay, I'm convinced. Why don't we be consistent and just add muscle to all
of the names?

Robert E. Druzinsky, Ph.D.
Clinical Associate Professor
Dept. of Oral Biology
College of Dentistry
University of Illinois at Chicago
801 S. Paulina
Chicago, IL 60612
druzinsk@uic.edu

Office: 312-996-0406
Lab: 312-996-0629
Website: www.peerj.com/RobertDruzinsky
